### ISSUE TITLE
Books don't work

### ISSUE THREAD

**Issue by gjorgdy** [TheLimeGlass/Skellett#103]
Whatever I do, I can't get the books to work, even with the example scripts I only get this:
![image](https://user-images.githubusercontent.com/36630987/87294460-220c6480-c504-11ea-952b-eb6c96e437d3.png)

**Comment by maciejpanocha** [comment]
Got the same problem.

**Comment by JerryTheSlime16** [comment]
This is since you have done the nbt of the book wrong.  Try setting a book in your hand to a variable & then open that variable.

**Comment by H0peT1me** [comment]
I have the same problem yet the code is correct :
```
command /opentest:
	trigger:
		set {_book} to new written book

		set {_text} to a new text component with text "&eHello World"
		add hover event with action SHOW_TEXT with text "&4GoodBye" to text component {_text}
		add text component {_text} to book {_book}
		open book {_book} to player #Don't work in book

		message text component {_text} to player #Work in chat
```

Screen :
[https://i.imgur.com/dH0TnTI.png](url)
[https://i.imgur.com/em6iRfl.png](url)

![2020-08-07_13 16 11](https://user-images.githubusercontent.com/68251816/89641483-0baeba00-d8b2-11ea-99c2-cfab47ed8559.png)
![2020-08-07_13 16 32](https://user-images.githubusercontent.com/68251816/89641481-0b162380-d8b2-11ea-9b15-253e5ddfdf16.png)


Ty

**Comment by hweh-png** [comment]
Set this to closed.

**Comment by Rqver** [comment]
Are we going to get a fix to this issue?

**Comment by TheLimeGlass** [comment]
Fixed in 2.0.0
